src/utils.py

```diff
@@ -4,7 +4,7 @@
 # TODO: make all functions work with strings as well
 # TODO: add a new cool calculator function
 
-def sum(a: int, b: int) -> int:
+def sum(a: Union[int, str], b: Union[int, str]) -> Union[int, str]:
```

Comment:
They only asked to make a function for to add integers and float, not string.

Reply:
Yes. I agree with this.